EIPS/eip-7044.md

```diff
@@ -0,0 +1,57 @@
+---
+eip: 7044
+title: Perpetually Valid Signed Voluntary Exits
+description: Lock voluntary exit signature domain on capella for perpetual validity
+author: Lion (@dapplion)
+discussions-to: https://ethereum-magicians.org/t/eip-7044-perpetually-valid-signed-voluntary-exits/14348
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-05-18
+---
+
+## Abstract
+
+Lock validator voluntary exit signature domain on Capella for perpetual validity. Currently, signed voluntary exits are only valid for two forks.
+
+## Motivation
+
+Currently, signed voluntary exits are valid up-to only two forks for block inclusion due to the Beacon Chain state considering only the current and previous fork. This limitation increases the complexity of some staking operations, specifically those that require the exchange of signed exits ahead of time for an unbounded number of forks.
+
+The limited validity of voluntary exits was originally motivated to isolate them in the event of a perpetual fork. If fork A and B exist and a validator operates on both, if they send an exit, it will be replayable on both. However, this possibility is not sufficient to justify the UX degradation exposed above.
```

Comment:
```suggestion
The limited validity of voluntary exits was originally motivated to isolate them in the event of a hard fork that results in two maintained chains. If fork A and B exist and a validator operates on both, if they send an exit, it will be replayable on both. However, this possibility is not sufficient to justify the UX degradation exposed above, as no funds are at risk and the staker can re-stake on one or both of the chains.
```